daemon/src/supervisor.rs

```diff
@@ -104,6 +120,29 @@ where
     }
 }
 
+#[async_trait]
+impl<T, R> xtra::Handler<Panicked> for Actor<T, R>
+where
+    T: xtra::Actor,
+    R: fmt::Display + 'static,
+{
+    async fn handle(&mut self, msg: Panicked, ctx: &mut Context<Self>) {
+        let actor = T::name();
+
+        match msg.error.downcast::<String>() {
+            Ok(reason) => {
+                tracing::error!("{actor} panicked: {reason}");
```

Comment:
Wouldn't it make sense to add the metrics to the logs so we see them?

Reply:
I am thinking about that yeah. Will push an update later!

Reply:
I opted for adding more structured logs, that should make it easier to compute such metrics downstream. Should I remove the metrics again? Only added them for testing really ..

Reply:
I opted for making these things module private for now.